src/wiki/putdoc.ts

```diff
@@ -0,0 +1,68 @@
+import { S3 } from 'aws-sdk';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+export const handler = async function (
+    event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> {
+    try {
+        const s3 = new S3({
+            accessKeyId: process.env.ACCESS_KEY,
+            secretAccessKey: process.env.SECRET_ACCESS_KEY,
+        });
+        const bucketName = process.env.BUCKET_NAME;
+
+        if (event === null) {
+            throw new Error('event not found');
+        }
+
+        if (!bucketName) {
+            throw new Error('Bucket not connected');
+        }
+
+        if (event.pathParameters === null) {
+            throw new Error('Request is missing params');
+        }
+
+        const { areaid, docid } = event.pathParameters;
+        if (!areaid || !docid) {
+            throw new Error('Request is missing params');
+        }
+
+        const trueArea = areaid.split(':').join('/');
```

Comment:
instead of allowing questionable areas names i think we should reject invalid names

for example u cant have a password with only numbers. please add a small logic where if the area includes special characters it should reject (excluding whitespace, underscore and dash)

plus another logic to "clean" the area name. if i want to make an area with name "my area with               so         many spaces", it should trim the whitespaces to one


for both area and doc